lib/event_sourceable/event_store.rb

```diff
@@ -13,7 +15,7 @@ class EventStore < ::ActiveRecord::Base
 
     after_initialize do
       self.data ||= {}
-      self.metadata ||= {}
+      self.metadata ||= EventSourceable.metadata
```

Comment:
Should this be `dup`'d?

Reply:
yes. let's dupe it just to prevent unintended changes by the event.